.goreleaser.yml

```diff
@@ -15,6 +15,29 @@ archives:
       - completions/*
       - manpages/*
 
+aurs:
+  - homepage:  https://www.updatecli.io
+    description: Updatecli is a declarative dependency management command line tool.
+    maintainers:
+      - 'Olivier Vernin <olivier at vernin dot me>'
+    license: 'Apache-2.0'
+    private_key: '{{ .Env.AUR_KEY }}'
+    git_url: 'ssh://aur@aur.archlinux.org/updatecli-bin.git'
+    package: |-
+      # bin
+      install -Dm755 "./updatecli" "${pkgdir}/usr/bin/updatecli"
```

Comment:
Use `${pkgname}` instead of the literal `updatecli`

Reply:
```suggestion
      install -Dm755 "./updatecli" "${pkgdir}/usr/bin/${pkgname}"
```